modules/web/js/ballerina/ast/defaults-added-ballerina-ast-factory.js

```diff
@@ -168,5 +168,20 @@ define(['lodash', './ballerina-ast-factory'], function (_, BallerinaASTFactory)
         return throwStatement;
     };
 
+    /**
+     * creates ForkJoinStatement
+     * @param args
+     */
+    DefaultsAddedBallerinaASTFactory.createForkJoinStatement = function (args) {
```

Comment:
What if we keep the same name in DefaultsAddedBallerinaASTFactory and BallerinaASTFactory, so that if user invoke "createForkJoinStatement" with DefaultsAddedBallerinaASTFactory.createForkJoinStatement it will reply a Forkjoin statement with defaults and if user invokes "createForkJoinStatement" with BallerinaASTFactory.createForkJoinStatement it will reply a empty Forkjoin statement
WDYT?